phpstan.neon

```diff
@@ -34,3 +36,6 @@ parameters:
         - '#Method ApiPlatform\\Core\\Bridge\\Doctrine\\Orm\\Filter\\FilterInterface::apply\(\) invoked with 5 parameters, 3-4 required\.#'
         - '#Method ApiPlatform\\Core\\Bridge\\Doctrine\\Orm\\Filter\\OrderFilter::filterProperty\(\) invoked with 7 parameters, 5-6 required\.#'
         - '#Method ApiPlatform\\Core\\DataProvider\\CollectionDataProviderInterface::getCollection\(\) invoked with 3 parameters, 1-2 required.#'
+
+        # Older versions of doctrine may not have the method
+        - '#Call to function method_exists\(\) with .* and .* will always evaluate to false\.#'
```

Comment:
Note that I have not managed to put a correct RegExp because of single quotes in the expectation:

```
        # - "#Call to function method_exists() with 'Doctrine\\\\ORM\\\\Query\\\\Expr\\\\Join' and 'getJoin' will always evaluate to false.#"
```

This should work but doesn't... This is the output I have:

```
Call to function method_exists() with 'Doctrine\\ORM\\Query…' and 'getJoin' will always evaluate to false.
```

Any ideas?